source/cache-proxy/setup-squid-as-a-transparent-proxy.rst

```diff
@@ -11,9 +11,11 @@ Install the Package
 
 First, install the Squid package.
 
-#. Click **System > Packages**
-#. Scroll down until the "squid" package listing is visible
-#. Click |fa-plus| located to the right of the squid package description
+#. Click **System > Package Manager**
+#. Click Available Packages 
```

Comment:
You should bold **Available Packages** (mouse input -- I'll add that to the style guide)